configure.ac

```diff
@@ -483,6 +477,10 @@ AC_SUBST([PART_LDFLAGS])
 CFLAGS=$SAVE_CFLAGS
 LDFLAGS=$SAVE_LDFLAGS
 
+
+# For disk notification and fault LED.
+PKG_CHECK_MODULES([LIBLSM], [libstoragemgmt >= 1.3.0])
```

Comment:
Thanks for the explanation, I didn't understand that this is supposed to be part of the core functionality not the LSM module. Anyway, ``have_lsm`` is not being set with these changes applied, is that okay (for the rest of the codebase)?